schedule/yast/minimal+base/minimal+base@yast_x86_64.yaml

```diff
@@ -1,31 +0,0 @@
----
-name:           minimal+base@yast
-description:    >
-  Select a minimal textmode installation by starting with the default and unselecting all patterns
-  except for "base" and "minimal". Not to be confused with the new system role "minimal" introduced with SLE15.
-vars:
-  DESKTOP: textmode
-  PATTERNS: base,enhanced_base
-  YUI_REST_API: 1
-schedule:
-  software:
-    - installation/select_only_visible_patterns_from_top
```

Comment:
how this unification can work if this module is what makes the system minimal?
https://openqa.suse.de/tests/15533634#step/select_only_visible_patterns_from_top/10

Reply:
From this module https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20157/files#diff-533c4d7653b2adbb069a6a75b6ec910955d0f017685b6d836105f268b9a66b8dR9

We select only the given ones here:
https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/327/diffs#b7e6307178e773596d52be14fee6d1d0e87718d4_924_926

Reply:
yes, we created the other one because was faster, didn't need to iterate over all the items, and make it more reliable, but I will need to see the verification to have an opinion. Your solution might just be fine, because that `select_only_visible_patterns_from_top` is very dependent of what is in the screen....the first pattern that appear afir.

Reply:
Yeah, on my way. The problem is minimal+base_yast test is failing in SP7, https://progress.opensuse.org/issues/166424

Probably we will need to solve this first.

Reply:
I will give VR for SP6 meanwhile

Reply:
please, use SP7, that is a post-installation failure, should not affect your verification runs.